lib/internal/util.js

```diff
@@ -5,19 +5,43 @@ const signals = process.binding('constants').os.signals;
 
 const kArrowMessagePrivateSymbolIndex = binding['arrow_message_private_symbol'];
 const kDecoratedPrivateSymbolIndex = binding['decorated_private_symbol'];
+const noCrypto = !process.versions.openssl;
 
-// The `buffer` module uses this. Defining it here instead of in the public
-// `util` module makes it accessible without having to `require('util')` there.
-exports.customInspectSymbol = Symbol('util.inspect.custom');
+/*
+ * Implementation of ToInteger as per ECMAScript Specification
+ * Refer: http://www.ecma-international.org/ecma-262/6.0/#sec-tointeger
+ */
+function toInteger(argument) {
+  const number = +argument;
+  return Number.isNaN(number) ? 0 : Math.trunc(number);
+}
 
-// Mark that a method should not be used.
-// Returns a modified function which warns once by default.
-// If --no-deprecation is set, then it is a no-op.
-exports.deprecate = function deprecate(fn, msg, code) {
+/*
+ * Implementation of ToLength as per ECMAScript Specification
+ * Refer: http://www.ecma-international.org/ecma-262/6.0/#sec-tolength
+ */
+function toLength(argument) {
+  const len = toInteger(argument);
+  return len <= 0 ? 0 : Math.min(len, Number.MAX_SAFE_INTEGER);
+}
+
+function isError(e) {
+  return objectToString(e) === '[object Error]' || e instanceof Error;
+}
+
```

Comment:
sometimes there is two lines beween definitions, sometimes one, should probably be consisten